VSRAD.Package/BuildTools/Errors/Parser.cs

```diff
@@ -20,11 +20,20 @@ public static ICollection<Message> ParseStderr(IEnumerable<string> outputs)
                     string line;
                     while ((line = reader.ReadLine()) != null)
                     {
-                        var message = ParseScriptMessage(line) ?? ParseKeywordMessage(line) ?? ParseClangMessage(line);
-                        if (message != null)
+                        var message = ParseKeywordMessage(line) ?? ParseClangMessage(line);
+                        var scriptMessages = ParseScriptMessage(line);
+                        if (message != null || scriptMessages != null)
                         {
-                            lastMessage = message;
-                            messages.Add(message);
+                            if (message != null)
+                            {
+                                lastMessage = message;
+                                messages.Add(message);
+                            }
+                            else
+                            {
+                                lastMessage = scriptMessages[scriptMessages.Count - 1];
+                                messages.AddRange(scriptMessages);
+                            }
                         }
                         else if (lastMessage != null)
```

Comment:
`lastMessage` is redundant I think, we can get rid of it if we change this condition to something like `if (messages.Count > 0) { messages.Last().Text += ... }`